apps/angular/src/app/features/anime/table/table.component.html

```diff
@@ -1,45 +1,67 @@
-<table
-	mat-table
-	*ngIf="animeList$ | async as animeList"
-	[dataSource]="animeList.results"
-	matSort
-	matSortActive="created"
-	matSortDisableClear
-	matSortDirection="desc"
-	aria-describedby="anime list"
->
-	<ng-container matColumnDef="poster">
-		<th class="poster-column__header" mat-header-cell *matHeaderCellDef>Image</th>
-		<td class="poster-column__poster-row" mat-cell *matCellDef="let row">
-			<img class="poster-column__poster" [src]="row.imageUrl" alt="Poster" />
-		</td>
-	</ng-container>
+<form [formGroup]="filterForm" (ngSubmit)="onSubmit()" class="filter-form">
+	<mat-form-field>
+		<mat-label>Type</mat-label>
+		<mat-select [formControl]="filterForm.controls.type" multiple>
+			<mat-option *ngFor="let filter of filters" [value]="filter">{{ filter }}</mat-option>
+		</mat-select>
+	</mat-form-field>
+	<mat-form-field>
+		<mat-label>Search</mat-label>
+		<input matInput placeholder="Ex. One Punch Man" [formControl]="filterForm.controls.search" />
```

Comment:
You can use just `formControlName` for controls inside form group.

```suggestion
		<input matInput placeholder="Ex. One Punch Man" formControlName="search" />
```